core/app/services/spree/export.rb

```diff
@@ -0,0 +1,103 @@
+require 'nokogiri'
+
+module Spree
```

Comment:
let's add some spec for this service object

Reply:
yeah, that's on my to-do list :)